### PR TITLE
[CPKAFKA-862] Fix Kafka startup instructions in quickstart

### PR DESCRIPTION
**Pull request by aayars**

### CODE REVIEW

**Comment by aayars** [comment]
Cc: @ewencp @arrawatia @coughman @cmccabe @maxzheng

**Comment by arrawatia** [comment]
I think we will need to add this property to all tutorials which spin up less than 3 brokers.

File: docs/tutorials/connect-avro-jdbc.rst [hunk 1]
```diff
@@ -63,12 +63,16 @@ Now that we have all of the Docker dependencies installed, we can create a Docke
...
         -e KAFKA_ZOOKEEPER_CONNECT=localhost:32181 \
         -e KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://localhost:29092 \
+        -e KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR=1 \
         confluentinc/cp-kafka:3.3.0-SNAPSHOT
 
   .. note::
 
     You'll notice that we set the ``KAFKA_ADVERTISED_LISTENERS`` variable to ``localhost:29092``.  This will make Kafka accessible from outside the container by advertising it's location on the Docker host.
 
+    We are also overriding offsets.topic.replication.factor to 1 at runtime, since there is only one active broker in this example.
```
Comment (ijuma) [inline docs/tutorials/connect-avro-jdbc.rst]: Do we need to add this sentence to the quickstart as well?

Comment (cmccabe) [inline docs/tutorials/connect-avro-jdbc.rst]: I added KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR=1 to a few more places.

**Pull request merged**